### PR TITLE
Create contributing guide for LFESS Working Groups

### PR DESCRIPTION
**Pull request by jmertic**
This PR captures both how to contribute to a working group/spec, along with best practices on how to run working groups effectively.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206477220263754

### CODE REVIEW

**Comment by jmertic** [comment]
@jorydotcom could you take a look and let me know if this looks accurate?

File: COLLABORATION_BEST_PRACTICES.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# LFESS Collaboration Best Practices
+
+This document outlines some of the key elements of meeting and collaboration guidance
+that is highly recommended for LFESS Working Groups to adopt.
+
+## Working Group Structure
+
```
Comment (jmertic) [inline COLLABORATION_BEST_PRACTICES.md]: Things to add

- What does it mean to be a WG member? Time commitments, expectations, etc.
- Specific roles and how they are selected

File: COLLABORATION_BEST_PRACTICES.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+common interest around the scope of the working group. Time is valuable, so it is 
+important to run meetings efficiently and effectively to enable collaboration and 
+ensure the specification produced is of high value.
+
+### Scheduling and Cadence
+
+### Agenda Management
+
+### Note-Taking and Action Items Management
+
```
Comment (jmertic) [inline COLLABORATION_BEST_PRACTICES.md]: Call out tooling ( hackmd, Google Docs to GH and wiki )

**Comment by jorydotcom** [comment]
So this looks like a pretty good start in terms of like, a scaffold/template for information - one thing you will be able to use, hopefully soon, is a training we're producing with LF Training. The other thing you will need to accomodate is that in some cases, spec advancement practices may vary depending on the patent mode selected. So you will want to refer back to the Membership Agreement/Working Group Charter. 

The last thought is, the term "Working Group" under LFESS refers entirely to the WGs that are chartered in the agreement. If a Working Group wants to have sub-structures we should encourage them to use "Task Force" "Tiger Team" etc.

**Pull request merged**